src/main/java/group1/mips_simulator/components/Computer.java

```diff
@@ -6,26 +6,26 @@
 import group1.mips_simulator.components.instructionExecution.ExecutionResult;
 import group1.mips_simulator.components.instructionExecution.InstructionExecutions;
 import group1.mips_simulator.components.instructionParts.Field;
-import group1.mips_simulator.components.instructionParts.instruction.Instruction;
-import group1.mips_simulator.components.instructionParts.instruction.InstructionFactory;
-import group1.mips_simulator.components.instructionParts.instruction.RXIA_Instruction;
-import group1.mips_simulator.components.instructionParts.instruction.Reg2RegInstruction;
+import group1.mips_simulator.components.instructionParts.instruction.*;
 import group1.mips_simulator.components.memParts.Memory;
+import group1.mips_simulator.components.memParts.Cache;
 
 
 /**
  * A Mips Computer is a class to represent the classical computer architecture
  * being discussed in class.
  */
 public class Computer {
-
-    public Memory memory;
-    public CPU cpu;
-    public ROM rom = new ROM();
+    //computer parts
+    public Memory memory;       // computer memory
+    public Cache cache;         // cache
```

Comment:
Why are we maintaining a cache and memory at this top level? I assumed that the Cache would "have-a" memory, so no need to have a direct reference to it again here. I guess it doesn't hurt, just make the chances of a future developer (me) from messing up and using the `memory` instead of `cache`